### ISSUE TITLE
Remove unused JavaScript

### ISSUE THREAD

**Issue by mgifford** [uswds/uswds#4293]
### Describe the bug

I just installed the USWDS locally, then ran Google Lighthouse to look at the performance. 

The report says that the transfer size is 163.2 KiB  and there is a potential savings of 93.0 KiB

This file still exists in the distribution: 
http://localhost:3000/themes/mandelbrot/js/mandelbrot.js?cachebust=1.9.4

But I'm confused as there was a PR to remove it that looks like it was approved:
https://github.com/uswds/uswds/pull/3274

Is mandelbrot.js needed by default? 

### Steps to reproduce the bug
1. Install locally
2. Go to http://localhost:3000/
3. Run Google Lighthouse

### Expected behaviour
Ideally we'd see a performance score of 100 with a base install. 

### Screenshots

<img width="1386" alt="Screen Shot of Google Lighthouse Results" src="https://user-images.githubusercontent.com/116832/129979924-f147e3dd-6a46-4455-a84a-e4339bef1fd6.png">

### System setup
- **USWDS version:** v2.12.0
- **Operating system:** MacOS
- **Browser and version:** Edge

### Additional context
Add any other context about the problem.

**Comment by mgifford** [comment]
This might be an upstream issue https://github.com/frctl/fractal

But still think that the USWDS default shouldn't have these problems.

**Comment by thisisdano** [comment]
Since this example is running the Fractal library site, you're going to see the Fractal library js included — which is the source of `mandelbrot.js`. This should not exist in the distribution javascript (`uswds.min.js`). That said, I'm sure there's still room to optimize the distribution package, and there's an opportunity to provide a more straightforward example page/site, as you outline in https://github.com/uswds/uswds/issues/4294 

And yes, we did disable the js in #3274, but we enabled it again as it introduced all sorts of usability and accessibility issues when disabled.
